pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -22,30 +22,49 @@
 , ninja
 , pkg-config
 , python3Packages
+, unzip
 , vulkan-loader
 , libXNVCtrl
+, wayland
 }:
 
-stdenv.mkDerivation rec {
+let
+  # Derived from subprojects/imgui.wrap
+  imgui = rec {
+    version = "1.81";
+    src = fetchFromGitHub {
+      owner = "ocornut";
+      repo = "imgui";
+      rev = "v${version}";
+      hash = "sha256-rRkayXk3xz758v6vlMSaUu5fui6NR8Md3njhDB0gJ18=";
+    };
+    patch = fetchurl {
```

Comment:
This should use fetchpatch.

Reply:
It's a zip, not a patch file. It's just referred to as a patch in https://github.com/flightlessmango/MangoHud/blob/master/subprojects/imgui.wrap.